### DISCUSSION TITLE
Training a custom model with span categorizater instead of NER

### DISCUSSION THREAD

**Discussion by Oumayma68** [explosion/spaCy#12985] [Help: Other Questions]
Since the NER component doesn't work with overlapping spans, I was wondering if it's possible to train a custom model with span categorizer instead of ner ? if it is possible how to do it?
For eg, I have this Sentence: "The launch of the new iPhone was a major event in the tech industry."
I want to extract the following entities/spans: 
"the new iPhone" [Product]
"The launch of the new iPhone" [Event]

**Comment by rmitsch** [comment]
Hi @Oumayma68, `spancat` is a trainable component. Check out this [example project](https://github.com/explosion/projects/tree/v3/tutorials/spancat_food_ingredients) (or some of the other publicly available resources).
